app/code/Magento/Catalog/Block/Adminhtml/Product/Helper/Form/Image.php

```diff
@@ -21,7 +21,7 @@ protected function _getUrl()
         $url = false;
         if ($this->getValue()) {
             $url = $this->_urlBuilder->getBaseUrl(
-                \Magento\Framework\UrlInterface::URL_TYPE_MEDIA
+                '_type' => \Magento\Framework\UrlInterface::URL_TYPE_MEDIA
```

Comment:
This code is not even syntactically correct: https://travis-ci.org/magento/magento2/jobs/167057903#L825

Did you mean array here?